src/Hydrators/ProjectHydrator.php

```diff
@@ -25,4 +27,30 @@ public static function getAllProjects(DatabaseConnection $db)
         }
         return $projectObjects;
     }
+
+    public static function getProjectById(DatabaseConnection $db, int $id)
+    {
+        $queryProjectAndClient = $db->prepare('SELECT projects.id, projects.name, client_id, clients.name AS client_name, clients.logo AS client_logo, deadline 
+                                                FROM projects INNER JOIN clients 
+                                                ON projects.client_id = clients.id 
```

Comment:
I don't think `client_id` needs to be qualified as `projects.client_id` as it's not ambiguous

Reply:
done